### PR TITLE
Here they are :D! 

### PR DESCRIPTION
**Pull request by zhijing-jin**
Hi Thiago,

Thank you for your encouragement. I see the significant contribution of your dataset to NLG, as well as your hard work to continuously curate this dataset. (I am touched to see that almost every week you make new commits.) I really like your dataset, and I am willing to contribute! Here are some careful manual fixes I spent quite a while to make:

1. Replace erraneous template names with correct ones
1. Fix the spaces around period (for better sentence tokenization)
1. Fix erraneous referrence to entities
1. Fix wrong punctuations (for better sentence tokenization)
1. Fix spelling errors
1. Fix word boundary

In case you want to reproduce them, here are some executable codes that I used to made these fixes. (Only the final fixes to delete empty sortedtriplets and sentences are done one by one on the files.)

``` bash
cd data/v1.5/en

# replace erraneous template names with correct ones
find . -type f -exec sed -i 's/AEGNT/AGENT/g' {} \;
find . -type f -exec sed -i 's/AAGENT-1/AGENT-1/g' {} \;
find . -type f -exec sed -i 's/AGENT-1(area/AGENT-1 (area/g' {} \;
find . -type f -exec sed -i 's/AGENT-1(has/AGENT-1 (has/g' {} \;
find . -type f -exec sed -i 's/AGENT-1(mass/AGENT-1 (mass/g' {} \;
find . -type f -exec sed -i 's/AGENT-1is/AGENT-1 is/g' {} \;
find . -type f -exec sed -i 's/BRUDGE-1/BRIDGE-1/g' {} \;
find . -type f -exec sed -i 's/BRIDGE-1which/BRIDGE-1 which/g' {} \;
find . -type f -exec sed -i 's/ RIDGE-2/ BRIDGE-2/g' {} \;
find . -type f -exec sed -i 's/PATIENTI-2/PATIENT-2/g' {} \;
find . -type f -exec sed -i 's/havePATIENT-3/have PATIENT-3/g' {} \;
find . -type f -exec sed -i 's/byPATIENT-1/by PATIENT-1/g' {} \;
find . -type f -exec sed -i 's/PATIENT-2-operated/PATIENT-2 -operated/g' {} \;

# fix the spaces around period (for better sentence tokenization)
find . -type f -exec sed -i 's/ PATIENT-1\.AGENT-1 / PATIENT-1 . AGENT-1 /g' {} \;
find . -type f -exec sed -i 's/ PATIENT-2\.AGENT-1 / PATIENT-2 . AGENT-1 /g' {} \;
find . -type f -exec sed -i 's/ PATIENT-4\.BRIDGE-1 / PATIENT-4 . BRIDGE-1 /g' {} \;

find . -type f -exec sed -i 's/ in 1907-07-11\.The Monocacy / in 1907-07-11. The Monocacy /g' {} \;
find . -type f -exec sed -i 's/  Fuad Masum(a kurdish politician)\.Kurdish is /  Fuad Masum (a kurdish politician). Kurdish is /g' {} \;
find . -type f -exec sed -i 's/ come from Indonesia \.Batagor  / come from Indonesia . Batagor  /g' {} \;
find . -type f -exec sed -i 's/ for NASA in 1963 .He  / for NASA in 1963. He  /g' {} \;

# fix erraneous referrence to entities
find . -type f -exec sed -i 's/The capital of BRIDGE-1 is PATIENT-2 and BRIDGE-2 largest city is PATIENT-1 \./The capital of BRIDGE-1 is PATIENT-2 and BRIDGE-1 largest city is PATIENT-1 ./g' {} \;

# fix wrong punctuations (for better sentence tokenization)
find . -type f -exec sed -i 's/Sour cream, chopped fruits, condensed milk\. granola, raisins and shredded coconut are the main ingredients in Bionico\./Sour cream, chopped fruits, condensed milk, granola, raisins and shredded coconut are the main ingredients in Bionico./g' {} \;
find . -type f -exec sed -i 's/September 1st\. 1969/September 1st, 1969/g' {} \;

# fix spelling errors
find . -type f -exec sed -i 's/ accademiz / academia /g' {} \;
find . -type f -exec sed -i 's/ unreleated / unrelated /g' {} \;
find . -type f -exec sed -i 's/ varation / variation /g' {} \;
find . -type f -exec sed -i 's/ variatons / variations /g' {} \;
find . -type f -exec sed -i 's/ oprated / operated /g' {} \;
find . -type f -exec sed -i 's/ originaly / originally /g' {} \;
find . -type f -exec sed -i 's/ origintes / originates /g' {} \;
find . -type f -exec sed -i 's/ poacea / poaceae /g' {} \;
find . -type f -exec sed -i 's/ posgraduayed / postgraduate /g' {} \;
find . -type f -exec sed -i 's/ prevously / previously /g' {} \;
find . -type f -exec sed -i 's/ publshed / published /g' {} \;
find . -type f -exec sed -i 's/ prevously / previously /g' {} \;
find . -type f -exec sed -i 's/ punlished / published /g' {} \;
find . -type f -exec sed -i 's/ recor / record /g' {} \;
find . -type f -exec sed -i 's/ relgiion / religion /g' {} \;
find . -type f -exec sed -i 's/ runwiay / runway /g' {} \;
find . -type f -exec sed -i 's/ sequeled / runway /g' {} \;
find . -type f -exec sed -i 's/ sppoken / spoken /g' {} \;
find . -type f -exec sed -i 's/ studiies / studies /g' {} \;
find . -type f -exec sed -i 's/ sytle / style /g' {} \;
find . -type f -exec sed -i 's/ tboh / both /g' {} \;
find . -type f -exec sed -i 's/ whic / which /g' {} \;
find . -type f -exec sed -i 's/ identfier / identifier /g' {} \;
find . -type f -exec sed -i 's/ idenitifier / identifier /g' {} \;
find . -type f -exec sed -i 's/ igredient / ingredients /g' {} \;
find . -type f -exec sed -i 's/ ingridient / ingredients /g' {} \;
find . -type f -exec sed -i 's/ inclusdes / includes /g' {} \;
find . -type f -exec sed -i 's/ indain / indian /g' {} \;
find . -type f -exec sed -i 's/ leaderr / leader /g' {} \;
find . -type f -exec sed -i 's/ legue / league /g' {} \;
find . -type f -exec sed -i 's/ lenght / length /g' {} \;
find . -type f -exec sed -i 's/ loaction / location /g' {} \;
find . -type f -exec sed -i 's/ locaated / located /g' {} \;
find . -type f -exec sed -i 's/ locatedd / located /g' {} \;
find . -type f -exec sed -i 's/ locationa / location /g' {} \;
find . -type f -exec sed -i 's/ manhattern / manhattan /g' {} \;
find . -type f -exec sed -i 's/ memberrs / members /g' {} \;
find . -type f -exec sed -i 's/ menbers / members /g' {} \;
find . -type f -exec sed -i 's/ meteres / metres /g' {} \;
find . -type f -exec sed -i 's/ numbere / number /g' {} \;
find . -type f -exec sed -i 's/ numberr / number /g' {} \;
find . -type f -exec sed -i 's/ abbreivated / abbreviated /g' {} \;
find . -type f -exec sed -i 's/ abreviated / abbreviated /g' {} \;
find . -type f -exec sed -i 's/ abreviation / abbreviation /g' {} \;
find . -type f -exec sed -i 's/ addres / address /g' {} \;
find . -type f -exec sed -i 's/ aerbaijan / azerbaijan /g' {} \;
find . -type f -exec sed -i 's/ azerbijan / azerbaijan /g' {} \;
find . -type f -exec sed -i 's/ affilaited / affiliated /g' {} \;
find . -type f -exec sed -i 's/ affliate / affiliate /g' {} \;
find . -type f -exec sed -i 's/ aircfrafts / aircraft /g' {} \;
find . -type f -exec sed -i 's/ aircarft / aircraft /g' {} \;
find . -type f -exec sed -i 's/ airpor / airport /g' {} \;
find . -type f -exec sed -i 's/ in augurated / inaugurated /g' {} \;
find . -type f -exec sed -i 's/ inagurated / inaugurated /g' {} \;
find . -type f -exec sed -i 's/ inaugrated / inaugurated /g' {} \;
find . -type f -exec sed -i 's/ ausitin / austin /g' {} \;
find . -type f -exec sed -i 's/ coccer / soccer /g' {} \;
find . -type f -exec sed -i 's/ comanded / commanded /g' {} \;
find . -type f -exec sed -i 's/ counrty / country /g' {} \;
find . -type f -exec sed -i 's/ countyof / county of /g' {} \;
find . -type f -exec sed -i 's/ creater / creator /g' {} \;
find . -type f -exec sed -i 's/ currecncy / currency /g' {} \;
find . -type f -exec sed -i 's/ denonym / demonym /g' {} \;
find . -type f -exec sed -i 's/ discipine / discipline /g' {} \;
find . -type f -exec sed -i 's/ engish / english /g' {} \;
find . -type f -exec sed -i 's/ ethinic / ethnic /g' {} \;
find . -type f -exec sed -i 's/ ethiopa / ethiopia /g' {} \;
find . -type f -exec sed -i 's/ ethipoia / ethiopia /g' {} \;
find . -type f -exec sed -i 's/ eceived / received /g' {} \;
find . -type f -exec sed -i 's/ ffiliated / affiliated /g' {} \;
find . -type f -exec sed -i 's/ grop / group /g' {} \;

# fix word boundary
find . -type f -exec sed -i 's/ withreference / with reference/g' {} \;
find . -type f -exec sed -i 's/ thememorial / the memorial /g' {} \;
find . -type f -exec sed -i 's/ establishedin / established in /g' {} \;
find . -type f -exec sed -i 's/ constructionof / construction of /g' {} \;
find . -type f -exec sed -i 's/ abbreviatedform / abbreviated form /g' {} \;
find . -type f -exec sed -i 's/ notablework / notable work /g' {} \;
find . -type f -exec sed -i 's/ managerof / manager of /g' {} \;
find . -type f -exec sed -i 's/ 7and / 7 and /g' {} \;
```

### CODE REVIEW

**Comment by ThiagoCF05** [comment]
Dear @zhijing-jin ,

That's great work! Thank you very much!

Instead of merging your changes into "master", I will do it in a separate branch called "v1.6", since some of the spelling errors you fixed (e.g., "# fix spelling errors") were made by the participants of the experiment in which WebNLG was collected. Removing these typos without keeping any reference to them may affect the originality of the dataset. So in this new branch, I will prepare the corpus in a way that I can keep a version with the spelling errors and another with your post-editions. Once I finish this preparation, I will merge this branch into master as the version 1.6 of the corpus. Is that ok? What do you think?

Once again, thank you very much for your contribution.

File: data/v1.5/en/train/3triples/University.xml [hunk 1]
```diff
@@ -196,9 +196,9 @@
 					<reference entity="&quot;Universitas Apulensis&quot;" number="3" tag="PATIENT-3" type="name">Universitas Apulensis</reference>
 					<reference entity="&quot;Breaz Valer Daniel&quot;" number="4" tag="PATIENT-2" type="name">Breaz Valer Daniel</reference>
 				</references>
-				<text>1 Decembrie 1918 University is known informally as Uab and the latin name is Universitas Apulensis. The recor is Breaz Valer Daniel.</text>
-				<template>AGENT-1 is known informally as PATIENT-1 and the latin name is PATIENT-3 . The recor is PATIENT-2 .</template>
-				<lexicalization>AGENT-1 VP[aspect=simple,tense=present,voice=passive,person=3rd,number=singular] know informally as PATIENT-1 and DT[form=defined] the latin name VP[aspect=simple,tense=present,voice=active,person=3rd,number=singular] be PATIENT-3 . DT[form=defined] the recor VP[aspect=simple,tense=present,voice=active,person=3rd,number=singular] be PATIENT-2 .</lexicalization>
+				<text>1 Decembrie 1918 University is known informally as Uab and the latin name is Universitas Apulensis. The record is Breaz Valer Daniel.</text>
```
Comment (Shimorina) [inline data/v1.5/en/train/3triples/University.xml:199]: should be "rector", not "record"

File: data/v1.5/en/train/5triples/WrittenWork.xml [hunk 2]
```diff
@@ -4742,8 +4742,8 @@
 					<reference entity="John_Cowper_Powys" number="5" tag="BRIDGE-3" type="name">John Cowper Powys</reference>
 					<reference entity="Shirley,_Derbyshire" number="6" tag="PATIENT-1" type="name">Shirley , Derbyshire</reference>
 				</references>
-				<text>A Glastonbury Romance was followed by Weymouth Sands, which is sequeled to Maiden Castle, and was preceded By Wolf Solent, which author is John Cowper Powys, who was born in Shirley, Derbyshire.</text>
-				<template>AGENT-1 was followed by BRIDGE-1 , which is sequeled to PATIENT-2 , and was preceded By BRIDGE-2 , which author is BRIDGE-3 , who was born in PATIENT-1 .</template>
+				<text>A Glastonbury Romance was followed by Weymouth Sands, which is runway to Maiden Castle, and was preceded By Wolf Solent, which author is John Cowper Powys, who was born in Shirley, Derbyshire.</text>
```
Comment (Shimorina) [inline data/v1.5/en/train/5triples/WrittenWork.xml:4745]: should be something else, but not "runway"

**Review by Shimorina — COMMENTED** [review]
Fix errors in misspellings

**Pull request merged**